src/connectionconfig/connectionconfig.ts

```diff
@@ -77,16 +63,14 @@ export class ConnectionConfig implements IConnectionConfig {
         };
 
         // Read from user settings
-        let parsedSettingsFile = this.readAndParseSettingsFile(ConnectionConfig.configFilePath);
-        let userProfiles = this.getProfilesFromParsedSettingsFile(parsedSettingsFile);
+        let userProfiles = this.getProfilesFromSettingsFile();
```

Comment:
Minor: remove `File` from the name since it's no longer a real file read, it's to/from an API that could store anywhere